Modules/_testcapimodule.c

```diff
@@ -4051,6 +4051,86 @@ raise_SIGINT_then_send_None(PyObject *self, PyObject *args)
 }
 
 
+static PyObject *
+test_pyobject_fastcall(PyObject *self, PyObject *args)
```

Comment:
"test_" is a prefix to avoid name conflict with the called function: "pyobject_fastcall" vs "_PyObject_FastCall". It avoids confusion when you have to debug _testcapi.